lib/ocpp/v201/charge_point.cpp

```diff
@@ -563,6 +589,113 @@ AuthorizeResponse ChargePoint::validate_token(const IdToken id_token, const std:
         return response;
     }
 
+    // C07: Authorization using contract certificates
+    if (id_token.type == IdTokenEnum::eMAID) {
+        // Temporary variable that is set to true to avoid immediate response to allow the local auth list
+        // or auth cache to be tried
+        bool tryLocalAuthListOrCache = false;
+        bool forwardToCsms = false;
```

Comment:
Nit: rename to `forwardedToCsms`. I now expected you to still take an action in the if statement checking this.